python/ray/tune/progress_reporter.py

```diff
@@ -706,8 +706,11 @@ def __init__(
             sort_by_metric=sort_by_metric,
         )
 
+    def _print(self, msg: str):
+        print(msg)
```

Comment:
Why do we need this? We never seem to overwrite this

And should we add this to tune reporter base instead?

Also, this seems to be static, so maybe just as a file-local function

Reply:
We don't override this, but it's a simple change to improve modularity in the classes, allowing a user/developer to subclass and use eg. `logger.info` instead of `print` without having to change all the code. I admit this is a bit of a drive-by change but it's a common pattern in many libraries and I don't see any downside to following it here as well.

It's not really applicable to TuneReporterBase imo as that class doesn't deal with actually displaying the messages, with the responsibility for that delegated to the subclasses.